docs/src/main/sphinx/functions/list-by-topic.rst

```diff
@@ -316,11 +316,15 @@ For more details, see :doc:`json`
 
 * :func:`is_json_scalar`
 * :func:`json_array_contains`
+* :ref:`json_array() <json_array>`
+* :func:`json_array_get`
```

Comment:
What's the order policy here? I thought `json_array` should be before `json_array_contains`.

Reply:
Oops .. fixed